kinesumer.go

```diff
@@ -35,8 +35,9 @@ const (
 
 // Config defines configs for the Kinesumer client.
 type Config struct {
-	App    string // Application name.
-	Region string // Region name. (optional)
+	App      string // Application name.
+	Region   string // Region name. (optional)
+	ClientID string // consumer group client id. (optional)
```

Comment:
Please use title case for comment: `Consumer group ...`

Reply:
oke noted